components/Card.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export default function card({ voc }) {
+  return (
+    <>
+      <StyledSection>
+        <p>{voc.word}</p>
+      </StyledSection>
+    </>
+  );
+}
+
+const StyledSection = styled.section`
+  border: 2px solid #ddd;
+  border-radius: 5px;
+  margin: 10px auto;
+  height: 100px;
+  max-width: 200px;
```

Comment:
Why did you choose absolute units and not relative units (like em, rem, vh, vw...)?

Reply:
Thanks for that hint!